app/components/user-status.js

```diff
@@ -53,4 +54,20 @@ export default class UserStatusComponent extends Component {
       ],
     },
   ];
+
+  @action async changeStatus(status) {
+    if (status === USER_STATES.ACTIVE) {
+      const updatedAt = Date.now();
+      const activeStateObj = {
```

Comment:
Can we name as `activeState` only as it's obvious that it's a object by reading the code